src/Arcus.EventGrid.Tests.Unit/Publishing/EventGridPublisherBuilderTests.cs

```diff
@@ -73,22 +70,20 @@ public void WithExponentialRetry_NegativeRetryCount_ShouldFailWithArgumentOutOfR
             const int negativeInt = -100;
 
             // Assert
-            Assert.Throws<ArgumentOutOfRangeException>(
-                () => CreateEventGridBuilder()
-                      .WithExponentialRetry<Exception>(negativeInt));
+            Assert.Throws<ArgumentOutOfRangeException>(() => CreateEventGridBuilder()
+                                                                .WithExponentialRetry<Exception>(negativeInt));
         }
 
         [Fact]
         public void WithCircuitBroker_PositiveCircuitBrokenDuration_ShouldCreatePublisher()
         {
             // Arrange
-            TimeSpan positiveInterval = TimeSpan.FromSeconds(5);
+            var positiveInterval = TimeSpan.FromSeconds(5);
 
             // Act
-            IEventGridPublisher publisher =
-                CreateEventGridBuilder()
-                    .WithCircuitBreaker<Exception>(10, positiveInterval)
-                    .Build();
+            var publisher =CreateEventGridBuilder()
```

Comment:
:) forgot a space.

Reply:
Our code styling tooling will fix this in the future, see #39